apis/management.cattle.io/v3/k8s_windows_default.go

```diff
@@ -409,3 +412,53 @@ func initWindows() {
 		}
 	}
 }
+
+func shouldIgnoreVersion(version string) bool {
+	// we stopped building Windows images at 1.12.10, so we ignore checking anything above 1.12.7 (which was the last release before 1.12.10)
+	versionParts := strings.Split(version, ".")
+	minor := versionParts[1]
+	if minor == "12" {
+		patchParts := strings.Split(versionParts[2], "-")
+		patch, err := strconv.Atoi(patchParts[0])
+		if err != nil {
+			panic("Can't convert string " + patchParts[0] + " to int")
+		}
+		if patch > 7 {
+			return true
+		}
+	}
+	// we stopped building Windows images at 1.13.9, so we ignore checking anything above 1.13.5 (which was the last release before 1.13.9)
+	if minor == "13" {
+		patchParts := strings.Split(versionParts[2], "-")
+		patch, err := strconv.Atoi(patchParts[0])
+		if err != nil {
+			panic("Can't convert string " + patchParts[0] + " to int")
+		}
+		if patch > 5 {
+			return true
+		}
+	}
+	// we stopped building Windows images at 1.14.5, so we ignore checking anything above 1.14.3 (which was the last release before 1.14.3)
```

Comment:
maybe this should be commented to indicate this is only applicable to the 2.2 line? in case someone looks at this and wonders what is going on in master/v2.3+ onwards